src/Akka.HealthCheck/Readiness/DefaultReadinessProbe.cs

```diff
@@ -17,28 +18,35 @@ namespace Akka.HealthCheck.Readiness
     /// </summary>
     public sealed class DefaultReadinessProbe : ReceiveActor
     {
+        private readonly ILoggingAdapter _log = Context.GetLogger();
         private readonly ReadinessStatus _readinessStatus;
         private readonly HashSet<IActorRef> _subscribers = new HashSet<IActorRef>();
 
         public DefaultReadinessProbe() : this(new ReadinessStatus(true, $"Live: {DateTimeOffset.UtcNow}"))
         {
+            _log.Debug("Rediness probe correctly configured.");
         }
 
         public DefaultReadinessProbe(ReadinessStatus readinessStatus)
         {
             _readinessStatus = readinessStatus;
 
-            Receive<GetCurrentReadiness>(_ => Sender.Tell(_readinessStatus));
+            Receive<GetCurrentReadiness>(_ => {
+                Sender.Tell(_readinessStatus);
+                _log.Debug("Rediness Status {0} reported", _readinessStatus);
```

Comment:
Does `ReadinessStatus` override its `ToString` method so the output is human-readable? Otherwise the default formatting may not be all that nice here.